### PR TITLE
Prepare release

### PR DESCRIPTION
**Pull request by MaxTru**
__Which issue does this PR address?__

Locally tested using https://github.com/bpmn-io/bpmn-js-examples/tree/master/properties-panel => works

Related to https://github.com/camunda/camunda-modeler/issues/2077

__Acceptance Criteria__

* [ ] Corresponds to the concept 
* [ ] Corresponds to the design 

__Definition of Done__

* [ ] corresponds to [the design principles](https://github.com/bpmn-io/design-principles)
* [ ] corresponds to [the code standards](https://github.com/bpmn-io/bpmn-js-properties-panel/blob/master/.github/CONTRIBUTING.md#creating-a-pull-request)
* [ ] passes Continuous Integration checks
* [ ] available as feature branch on GitHub
  * [ ] contains the cleaned up commit history
  * [ ] commit messages satisfy our [commit message guidelines](https://www.conventionalcommits.org/)
  * [ ] a single commit closes the issue via Closes #issuenr

### CODE REVIEW

**Comment by barmac** [comment]
How about [this](https://github.com/bpmn-io/bpmn-js-properties-panel/commit/996046e92d6ae2049ce7149966c54e78e5aeb1f9)?

**Comment by MaxTru** [comment]
> How about [this](https://github.com/bpmn-io/bpmn-js-properties-panel/commit/996046e92d6ae2049ce7149966c54e78e5aeb1f9)? It was done after 0.39.0 so I think it's also worth including in the changelog.

True - thanks

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,10 +5,14 @@ All notable changes to [bpmn-js-properties-panel](https://github.com/bpmn-io/bpm
 ## Unreleased
 
 ___Note:__ Yet to be released changes appear here._
-* `CHORE`: bump to `bpmn-js@8.2.0`
-
 ...
```
Comment (barmac) [inline CHANGELOG.md:8]: We miss a newline here so it looks good again:

image

Comment (MaxTru) [reply CHANGELOG.md:9]: :facepalm: thanks you

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,10 +5,15 @@ All notable changes to [bpmn-js-properties-panel](https://github.com/bpmn-io/bpm
 ## Unreleased
 
 ___Note:__ Yet to be released changes appear here._
-* `CHORE`: bump to `bpmn-js@8.2.0`
 
 ...
 
+## 0.40.0
+
+* `CHORE`: bump to @bpmn-io/extract-process-variables@0.4.0
```
Comment (barmac) [inline CHANGELOG.md:13]:
```suggestion
* `CHORE`: bump to `@bpmn-io/extract-process-variables@0.4.0`
```

**Review by barmac — COMMENTED** [review]
One more comment

**Review by barmac — APPROVED** [review]

**Pull request merged**